scripts/check_enterprise_imports.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# This file checks all our AGPL licensed source files to be sure they don't
+# import any enterprise licensed packages (the inverse is fine).
+
+set -euo pipefail
+# shellcheck source=scripts/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")/lib.sh"
+cdroot
+
+set +e
+find . -regex ".*\.go" | grep -v "./enterprise" | xargs grep -n "github.com/coder/coder/enterprise"
+# reverse the exit code because we want this script to fail if grep finds anything.
+status=$?
+set -e
+if [ $status -eq 0 ]; then
+	echo "AGPL code cannot import enterprise!"
+	exit 1
```

Comment:
```suggestion
	error "AGPL code cannot import enterprise!"
```

Reply:
clearly my bash is pretty weak.  Thanks.